packages/applications/ssr/src/components/organisms/corrigerDocument/CorrigerDocumentModalForm.tsx

```diff
@@ -0,0 +1,71 @@
+'use client';
+
+import Button from '@codegouvfr/react-dsfr/Button';
+import { useState } from 'react';
+
+import { ModalWithForm } from '@/components/molecules/ModalWithForm';
+
+import { FormProps } from '../../atoms/form/Form';
+import { UploadDocument } from '../../atoms/form/UploadDocument';
+
+import { corrigerDocumentAction } from './corrigerDocument.action';
+
+type CorrigerDocumentModalFormProps = {
+  documentKey: string;
+  title?: string;
+  buttonLabel?: string;
+  confirmationLabel?: string;
+  uploadDocumentLabel?: string;
+  onSuccess?: FormProps['onSuccess'];
+};
+
+const defaultButtonLabel = 'Corriger';
+const defaultTitle = 'Corriger le document';
+const defaultConfirmationLabel = 'Êtes-vous sûr de vouloir corriger le document ?';
+const defaultUploadDocumentLabel = 'Fichier corrigé';
+
+export const CorrigerDocumentModalForm = ({
+  documentKey,
+  title = defaultTitle,
+  buttonLabel = defaultButtonLabel,
+  confirmationLabel = defaultConfirmationLabel,
+  uploadDocumentLabel = defaultUploadDocumentLabel,
+  onSuccess,
+}: CorrigerDocumentModalFormProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <>
+      <Button priority="secondary" onClick={() => setIsOpen(true)} style={{ marginTop: 0 }}>
```

Comment:
peut-être plus consistant d'utiliser tailwind plutot que `style`? 

```suggestion
      <Button priority="secondary" onClick={() => setIsOpen(true)} className="mt-0">
```

Reply:
Il y a un override côté DSF si je ne passe pas par style malheureusement

Reply:
ah d'accord pas de soucis